Makefile

```diff
@@ -123,8 +123,7 @@ v8:
 test: all
 	$(MAKE) build-addons
 	$(MAKE) cctest
-	$(PYTHON) tools/test.py --mode=release -J \
-		addons doctool inspector known_issues message pseudo-tty parallel sequential
+	$(PYTHON) tools/test.py --mode=release -J $(CI_JS_SUITES)
```

Comment:
@richardlau I included that now. Can you please check if the batch file changes are okay?